aquacomputer_d5next.c

```diff
@@ -247,15 +247,18 @@ static u16 aquastreamult_sensor_fan_offsets[] = { AQUASTREAMULT_FAN_OFFSET };
 #define OCTO_NUM_FANS			8
 #define OCTO_NUM_SENSORS		4
 #define OCTO_NUM_VIRTUAL_SENSORS	16
+#define OCTO_NUM_FLOW_SENSORS		1
 #define OCTO_CTRL_REPORT_SIZE		0x65F
 
 /* Sensor report offsets for the Octo */
 #define OCTO_SENSOR_START		0x3D
 #define OCTO_VIRTUAL_SENSORS_START	0x45
+#define OCTO_FLOW_SENSOR_OFFSET	0x7B
 static u16 octo_sensor_fan_offsets[] = { 0x7D, 0x8A, 0x97, 0xA4, 0xB1, 0xBE, 0xCB, 0xD8 };
 
 /* Control report offsets for the Octo */
 #define OCTO_TEMP_CTRL_OFFSET		0xA
+#define OCTO_FLOW_PULSES_CTRL_OFFSET	0x6
```

Comment:
Okay, I believe the control register offset is correct then, since I'm getting the same sysfs reading as in the official Aquacomputer app?

Good catch with missing the hwmon fan entry :+1:

Reply:
> Okay, I believe the control register offset is correct then, since I'm getting the same sysfs reading as in the official Aquacomputer app?

Then that should be it 👍 Try setting it to an allowed value and see if it reflects in Aquasuite.

Reply:
I verified this by changing sensor type in the app (e.g. `Aqua Computer high flow 2` -> `144`, `Digmesa 3.3mm` -> `509`) and the correct pulses value shows up in sysfs too.

Would it make sense to document the values that are getting set by the app while we're at it? ie what the pulses register should be set to for various sensor types?

Reply:
> Would it make sense to document the values that are getting set by the app while we're at it? ie what the pulses register should be set to for various sensor types?

That sounds a bit out of scope, IIUC. If it's something simple, maybe, but anything other than "hey, this refers to the flow ticks per liter" should refer the user to the official docs/manual. It's device/sensor dependent, after all, right?

Reply:
OK, I will limit the scope of this PR and only document the new registers then (commits added!)